Components/SettingsBar/SettingsSearch.tsx

```diff
@@ -0,0 +1,15 @@
+const SettingsSearch = () => {
```

Comment:
rename the file name to just `Search.tsx`

Reply:
ok sir
https://github.com/PiyushKalyanpy/GyanaGuru/pull/264/commits/5d4545e052a0dd832f07e5407a00997606625223